tests/test_executor.py

```diff
@@ -401,6 +401,22 @@ def test_execute_subqueries(self):
             ],
         )
 
+        schema = {"table1": {"type": "str", "id": "str", "parent_id": "str", "sub_type": "str"}}
+
+        executed = execute(
+            """
+            WITH cte1 AS (
+                SELECT id AS id_alias, sub_type
+                FROM (SELECT id, sub_type FROM table1)
+            ) SELECT * FROM cte1
+            """,
```

Comment:
In order to really test this scenario, we should generate an AST here, not using SQL

Reply:
Thanks!